### PR TITLE
Add student (congtinh) image and info

### PR DESCRIPTION
**Pull request by dcongtinh**
Please see the result at https://dcongtinh.github.io/gci16.fossasia.org/

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -221,3 +221,10 @@
   image: m1guelpiedrafita.png
   blog: blog.miguelpiedrafita.com
   bio: An spanish coder that loves open source.
+  
+- name: Tinh Dao
+  github: dcongtinh
+  image: tinhdao.jpg
+  twitter: TinhDao3
+  bio: 17, student, competitive programmer, interested in games
+  facebook: dcongtinh
```
Comment (hoangvanthien) [inline _data/students.yml]: Please add an empty line at the end of the file.

**Review by jigyasa-grover — CHANGES_REQUESTED** [review]
Also provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.

**Comment by dcongtinh** [comment]
I've updated the description. Please see for more details.

**Review by jigyasa-grover — APPROVED** [review]

**Comment by jigyasa-grover** [comment]
@dcongtinh Because of adding another student's work there are merge conflicts in this. Resolve merge conflicts, this [How To Manually Fix Git Merge Conflicts (video)](https://www.youtube.com/watch?v=g8BRcB9NLp4) might be helpful.

**Pull request merged**